src/app/service/eco-news/create-eco-news.service.ts

```diff
@@ -0,0 +1,67 @@
+import { Injectable } from '@angular/core';
+import { CreateNewsInterface } from '../../component/general/eco-news/create-news/create-news-interface';
+import { HttpClient, HttpHeaders } from '@angular/common/http';
+import { Observable } from 'rxjs';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class CreateEcoNewsService {
+
+  constructor(private http: HttpClient) { }
+
+  public translations: object = {
+    'ua': {
+      text: "",
+      title: ""
+    },
+    'ru': {
+      text: "",
+      title: ""
+    },
+    'en': {
+      text: "",
+      title: ""
+    }
+  };
+
+  private url: string = 'https://greencity.azurewebsites.net/econews';
+  private accessToken: string = localStorage.getItem('accessToken');
+  private httpOptions: any = {
```

Comment:
You can delete type declaration here, 'cause you create new HttpHeaders instance

Reply:
fixed